workflows/phaseimpute/main.nf

```diff
@@ -195,6 +222,16 @@ workflow PHASEIMPUTE {
                     ch_chunks = CHUNK_PREPARE_CHANNEL(ch_chunks, "glimpse").out.chunks
```

Comment:
Just `CHUNK_PREPARE_CHANNEL(ch_chunks, "glimpse").out.chunks` should be enough no ?

Reply:
We need to assign it to `ch_chunks` which is what `VCF_IMPUTE_GLIMPSE2` uses.